src/open_producten/producten/models/eigenaar.py

```diff
@@ -0,0 +1,97 @@
+from django.core.exceptions import ValidationError
+from django.core.validators import MinLengthValidator, RegexValidator
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from open_producten.producten.models.validators import validate_bsn
+from open_producten.utils.models import BaseModel
+
+from .product import Product
+
+
+class Eigenaar(BaseModel):
+
+    product = models.ForeignKey(
```

Comment:
I probably don't understand the use case, but this being a foreign key does mean they can't be reused and we'll end up with a bunch of duplicated Eigenaren if multiple Producten have the same eigenaar (with the same BSN that is).

How is this feature intended to be used? And what's the use of splitting it off of Product if the Eigenaren can't be reused across multiple producten? @alextreme

Reply:
Expanding upon Floris' explanation this is comparable to Rol/Betrokkene in Open Zaak, with the distinction that Eigenaar has a single 'roltype'. Scoping an Eigenaar to a single Product allows an API client to easily modify an Eigenaar